Blazor routing fallbacks for dots in params (#19764)
dotnet/AspNetCore.Docs: aspnetcore/blazor/fundamentals/routing.md

### Routing with URLs that contain dots

> For hosted Blazor WebAssembly and Blazor Server apps, the server-side default route template assumes that if the last segment of a request URL contains a dot (`.`) that a file is requested (for example, `https://localhost.com:5001/example/some.thing`). Without additional configuration, an app returns a *404 - Not Found* response for a static file that doesn't exist. To use a route with one or more parameters that contains a dot, the app must configure the route with a custom template.

Reviewer: ```suggestion
For hosted Blazor WebAssembly and Blazor Server apps, the server-side default route template assumes that if the last segment of a request URL contains a dot (`.`) that a file is requested (for example, `https://localhost.com:5001/example/some.thing`). Without additional configuration, an app returns a *404 - Not Found* response if this was meant to route to a component. To use a route with one or more parameters that contains a dot, the app must configure the route with a custom template.
```

Reviewer: Something like that? I'm not sure why the static file is mentioned there
